schedule-view/dist/assets/index-a91f7dc8.js

```diff
@@ -0,0 +1,5 @@
ass:Jt(["day",{"day-disabled":!o.active}])},dt(o.date),3))),256)),(ye(!0),xe(se,null,an(i.createTaskList,o=>(ye(),xe("section",{class:"task task--warning",style:zt({"grid-row":o.row+2+"/span 1","grid-column":o.column+1+"/span "+o.length})},[st("span",null,dt(o.data),1)],4))),256))])}const xl=Ts(ml,[["render",yl],["__scopeId","data-v-b7f5aa17"]]);const wl={data(){return{monthNames:["January","February","March","April","May","June","July","August","September","October","November","December"]}}},Is=e=>(Wi("data-v-42f8e8b2"),e=e(),Vi(),e),El={class:"calender-controls"},Ol=Is(()=>st("div",{class:"prev"},"<",-1)),Cl={class:"date"},Tl=Is(()=>st("div",{class:"prev"},">",-1));function Il(e,t,n,r,s,i){return ye(),xe("div",El,[Ol,st("div",Cl,dt(s.monthNames[this.$store.getters.selectedMonth])+" "+dt(this.$store.getters.selectedYear),1),Tl])}const Ml=Ts(wl,[["render",Il],["__scopeId","data-v-42f8e8b2"]]);const Pl={class:"main"},Al={__name:"App",setup(e){return(t,n)=>(ye(),xe("div",Pl,[we(Ml),we(xl)]))}};function Sl(){return Ms().__VUE_DEVTOOLS_GLOBAL_HOOK__}function Ms(){return typeof navigator<"u"&&typeof window<"u"?window:typeof global<"u"?global:{}}const Fl=typeof Proxy=="function",jl="devtools-plugin:setup",Nl="plugin:settings:set";let Ye,An;function $l(){var e;return Ye!==void 0||(typeof window<"u"&&window.performance?(Ye=!0,An=window.performance):typeof global<"u"&&(!((e=global.perf_hooks)===null||e===void 0)&&e.performance)?(Ye=!0,An=global.perf_hooks.performance):Ye=!1),Ye}function Dl(){return $l()?An.now():Date.now()}class Rl{constructor(t,n){this.target=null,this.targetQueue=[],this.onQueue=[],this.plugin=t,this.hook=n;const r={};if(t.settings)for(const o in t.settings){const l=t.settings[o];r[o]=l.defaultValue}const s=`__vue-devtools-plugin-settings__${t.id}`;let i=Object.assign({},r);try{const o=localStorage.getItem(s),l=JSON.parse(o);Object.assign(i,l)}catch{}this.fallbacks={getSettings(){return i},setSettings(o){try{localStorage.setItem(s,JSON.stringify(o))}catch{}i=o},now(){return Dl()}},n&&n.on(Nl,(o,l)=>{o===this.plugin.id&&this.fallbacks.setSettings(l)}),this.proxiedOn=new Proxy({},{get:(o,l)=>this.target?this.target.on[l]:(...u)=>{this.onQueue.push({method:l,args:u})}}),this.proxiedTarget=new Proxy({},{get:(o,l)=>this.target?this.target[l]:l==="on"?this.proxiedOn:Object.keys(this.fallbacks).includes(l)?(...u)=>(this.targetQueue.push({method:l,args:u,resolve:()=>{}}),this.fallbacks[l](...u)):(...u)=>new Promise(a=>{this.targetQueue.push({method:l,args:u,resolve:a})})})}async setRealTarget(t){this.target=t;for(const n of this.onQueue)this.target.on[n.method](...n.args);for(const n of this.targetQueue)n.resolve(await this.target[n.method](...n.args))}}function Ll(e,t){const n=e,r=Ms(),s=Sl(),i=Fl&&n.enableEarlyProxy;if(s&&(r.__VUE_DEVTOOLS_PLUGIN_API_AVAILABLE__||!i))s.emit(jl,e,t);else{const o=i?new Rl(n,s):null;(r.__VUE_DEVTOOLS_PLUGINS__=r.__VUE_DEVTOOLS_PLUGINS__||[]).push({pluginDescriptor:n,setupFn:t,proxy:o}),o&&t(o.proxiedTarget)}}/*!
```

Comment:
Should we be committing the `dist/` directory? Would it be better for someone to do that locally using `spin build`?